app.js

```diff
@@ -0,0 +1,16 @@
+const express = require("express")
+const {getAllTopics} = require("./controllers/topics.controller")
+const {applicationErrors} = require("./errors/error-handling")
+
+const app = express()
+app.use(express.json())
```

Comment:
your code isn't using this line yet, I would take it out until the code requires you to put it in